packages/notebook-extension/schema/panel.json

```diff
@@ -18,7 +18,7 @@
       { "name": "restart", "command": "kernelmenu:restart", "rank": 32 },
       {
         "name": "restart-and-run",
-        "command": "runmenu:restart-and-run-all",
+        "command": "notebook:restart-run-all",
```

Comment:
Would you mind adding a small note in the migration guide?

If people have customized the toolbar, this may change their settings.

Reply:
Thanks — I updated the migration guide with a note.